rclcpp/include/rclcpp/allocator/allocator_common.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCLCPP_RCLCPP__ALLOCATOR_COMMON__HPP_
+#define RCLCPP_RCLCPP__ALLOCATOR_COMMON__HPP_
+
+#include <memory>
+
+#include <rclcpp/allocator/allocator_deleter.hpp>
+
+namespace rclcpp
+{
+namespace allocator
+{
+
+template<typename U, typename Alloc>
```

Comment:
What does the `U` stand for? If nothing specific I would recommend to use the common variable `T` for template typenames.

Same below.